sdks/java/io/kafka/build.gradle

```diff
@@ -64,4 +69,24 @@ dependencies {
   testCompile library.java.powermock_mockito
   testRuntimeOnly library.java.slf4j_jdk14
   testRuntimeOnly project(path: ":runners:direct-java", configuration: "shadow")
+  kafkaVersion210 "org.apache.kafka:kafka-clients:2.1.0"
+}
+
+configurations.kafkaVersion210 {
+  resolutionStrategy {
+    force "org.apache.kafka:kafka-clients:2.1.0"
+  }
+}
+
+task kafkaVersion210Test(type: Test) {
+  group = "Verification"
+  description = 'Runs KafkaIO tests with Kafka clients API 2.1.0'
+  outputs.upToDateWhen { false }
```

Comment:
Instead of doing `outputs.upToDateWhen { false }` to have this run every time, please include the set of inputs/outputs that makes sense in a follow-up PR so that an incremental build can be supported.